Commands/Public/room.js

```diff
@@ -0,0 +1,195 @@
+const ArgParser = require("../../Modules/MessageUtils/Parser");
+const moment = require("moment");
+
+module.exports = async ({ Constants: { Colors, Text }, client }, { serverDocument, serverQueryDocument }, msg, commandData) => {
+	const roomDocument = serverDocument.config.room_data.id(msg.channel.id);
+	if (roomDocument) {
+		const question = await msg.send({
+			embed: {
+				color: Colors.PROMPT,
+				description: `This room was created ${moment(roomDocument.created_timestamp).fromNow()}. Would you like to delete it?`,
+			},
+			footer: {
+				text: "You have 1 minute to respond.",
+			},
+		});
+		let response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+		if (response) {
+			try {
+				await response.delete();
+			} catch (err) {
```

Comment:
Remove `(err)`

Reply:
Using that syntax would require our ESLint configuration to be modified to use a newer ECMAScript parser, and I'm not sure we want to do that yet. Additionally, it would force a lot of maintainers to update their node to very recent versions.